packages/vite/src/node/__tests__/dev.spec.ts

```diff
@@ -17,3 +20,62 @@ describe('resolveBuildEnvironmentOptions in dev', () => {
     expect(config.build.rollupOptions).not.toHaveProperty('input')
   })
 })
+
+describe('the dev server', () => {
+  let server: ViteDevServer
+
+  afterEach(() => {
+    server?.close()
+  })
+
+  test.each([
+    [
+      {
+        port: 5013,
+      },
+      {
+        local: ['http://localhost:5013/'],
+        network: [],
+      },
+    ],
+    [
+      {
+        port: 5014,
+        host: '127.0.0.1',
+      },
+      {
+        local: ['http://127.0.0.1:5014/'],
+        network: [],
+      },
+    ],
+  ])('resolves the server URLs for %s', async (options, expectation) => {
```

Comment:
```suggestion
  test('server.resolvedUrls exists at the time of `listening` event', async (options, expectation) => {
```
Since the purpose here is to test that `server.resolvedUrls` exists at the time of the `listening` event, I think testing just the first case is sufficient. Could you remove the second one?

Reply:
Thanks for the feedback. You are right, at this point it only matters to test if any host was resolved.